specification/security/resource-manager/readme.md

```diff
@@ -78,22 +78,33 @@ These are the global settings for the Security API.
 title: SecurityCenter
 description: API spec for Microsoft.Security (Azure Security Center) resource provider
 openapi-type: arm
-tag: package-composite-v3
+tag: package-preview-2023-02
```

Comment:
And we assume the default tag should contains all apis, but since you have changed the default tag to `package-preview-2023-02` is this expected?

Reply:
This change was made by the OpenAPI automation.